### ISSUE TITLE
Optimisation idea

### ISSUE THREAD

**Issue by kaihendry** [kaihendry/graphsloc#2]
Every commit we recount everything.

It would be a lot faster just to do a relative calculation using the sum of additions and deletions whilst iterating through the commits.

Of course I don't think SAMPLE would work very well here.

**Comment by kaihendry** [comment]
```
git diff --numstat 4115288b4f7bbd694946a1ddef0f0ba85c03f9a1 359be4460e89e325fb2969b1f231c9507d1c116b | awk 'NF==3 {plus+=$1; minus+=$2} END {prf(plus-minus)}'
```

**Comment by kaihendry** [comment]
Fixed in e6feee3a30c971e5d07d5623cc6ee187a1771f3f
